pkg/execute/factory.go

```diff
@@ -56,7 +57,11 @@ func NewExecutorFactory(log logrus.FieldLogger, cmdRunner CommandRunner, cfg con
 }
 
 // NewDefault creates new Default Executor.
-func (f *DefaultExecutorFactory) NewDefault(platform config.CommPlatformIntegration, notifierHandler NotifierHandler, isAuthChannel bool, conversationID string, bindings []string, message string) Executor {
+func (f *DefaultExecutorFactory) NewDefault(platform config.CommPlatformIntegration, notifierHandler NotifierHandler, isAuthChannel bool, conversationID string, bindings []string, message string, user ...string) Executor {
```

Comment:
In general it's not popular to use the variadic arguments as this is always the only property that can be specified. So in normal situation you should do just the `, user string)`.